### DISCUSSION TITLE
Date picker, multi-select dropdown components are missing.

### DISCUSSION THREAD

**Discussion by Umeshunp** [chakra-ui/chakra-ui#6254] [General]
### Description

It would be really useful if we get the date range picker, multi select dropdown components.

### Problem Statement/Justification

We will be using date range picker and multi select components in our most of the forms.
Only for few missing components we need to find  alternative libraries or write our own custom components.

It will be very useful if we get these components as part of chakra-ui.

### Proposed Solution or API

Provide date range picker & multi select dropdown components.

**Comment by shigetaichi** [comment]
@Umeshunp 
FYI: Multi-select component with chakra design currently available on here→ https://github.com/csandman/chakra-react-select 💡

**Reply by csandman** [reply]
Thanks a lot for sharing my package! I've put a lot of time into it, and it's nice to see people are getting good use from it!

**Comment by moussa32** [comment]
> @Umeshunp FYI: Multi-select component with chakra design currently available on here→ [https://github.com/csandman/chakra-react-select💡](https://github.com/csandman/chakra-react-select%F0%9F%92%A1)

Thank you for sharing, also the emoji in the end of your message is ruining the url

**Comment by NickVishwamitra** [comment]
**[From Chakra's official docs](https://v1.chakra-ui.com/faq#is-there-a-datepicker-in-chakra-ui):**
> No, currently we don't provide a datepicker component. We recommend building your own custom datepicker with Chakra UI and either react-datepicker or dayzed.

**Comment by Umeshunp** [comment]
> @Umeshunp FYI: Multi-select component with chakra design currently available on here→ https://github.com/csandman/chakra-react-select 💡

@shigetaichi Finally I have used this library, It would be nice if we have those components under chakra-ui itself.

**Reply by csandman** [reply]
As the creator of that package, I agree that it would be nice to have a native component! But alas, they haven't even started development on a Multi-Select yet: https://chakra-ui.canny.io/
